test/unit/TicketBroker.js

```diff
@@ -1735,4 +1735,93 @@ contract("TicketBroker", accounts => {
+        })
+        it("Returns claimable reserve for a claimaint if reserve is not frozen", async () => {
+            const numRecipients = 10
+            const deposit = 1000
+            const reserve = 1000
+            await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound)
+            await fixture.bondingManager.setMockUint256(functionSig("getTranscoderPoolSize()"), numRecipients)
+            await fixture.bondingManager.setMockBool(functionSig("isRegisteredTranscoder(address)"), true)
+            await broker.fundDepositAndReserve(deposit, reserve, {from: sender, value: deposit+reserve})
+            assert.equal(
+                (await broker.claimableReserve(sender, recipient)).toString(10),
+                (reserve/numRecipients).toString(10)
+            )
+            const recipientRand = 5
+            const faceValue = 10
+            const ticket = createWinningTicket(recipient, sender, recipientRand, faceValue)
+            const senderSig = await web3.eth.sign(getTicketHash(ticket), sender)
+            // Redeem a winning ticket
+            await broker.redeemWinningTicket(ticket, senderSig, recipientRand, {from: recipient})
+            // Reserve allocated for recipient should still be 100 since ticket was drawn from deposit 
+            assert.equal(
+                (await broker.claimableReserve(sender, recipient)).toString(10),
+                "100"
+            )
+            // Ticket faceValue should be substracted from deposit
+            assert.equal(
+                (deposit-faceValue).toString(10),
+                (await broker.getSenderInfo(sender)).sender.deposit.toString(10)
+            )
+        })
+        it("Returns claimable reserve for a claimant after reserve has been frozen", async () => {
+            const numRecipients = 10
+            const reserve = 1000
+            await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound)
+            await fixture.bondingManager.setMockUint256(functionSig("getTranscoderPoolSize()"), numRecipients)
+            await fixture.bondingManager.setMockBool(functionSig("isRegisteredTranscoder(address)"), true)
+            await broker.fundReserve({from: sender, value: reserve})
+
+            const recipientRand = 5
+            const faceValue = 10
+            const ticket = createWinningTicket(recipient, sender, recipientRand, faceValue)
+            const senderSig = await web3.eth.sign(getTicketHash(ticket), sender)
+            // Claim winning ticket - will freeze reserve (deposit = 0)
+            await broker.redeemWinningTicket(ticket, senderSig, recipientRand, {from: recipient})
+            // Assert reserve is frozen
+            await expectRevertWithReason(
+                broker.fundDeposit({from: sender, value: 1000}),
+                "sender's reserve is frozen"
+            )
+            // claimableReserve should be equal to reserve/numRecepients - faceValue
```

Comment:
`numRecpients` -> `numRecipients`

Reply:
I'll see if there's a spellchecker for vsCode to mitigate these types of mistakes in the future.